tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,59 @@
+"""Task 1 of Module 3."""
+import pyinputplus as pyip
+
+BREAD = {'wheat': 1.321, 'white': 2.5, 'sourdough': 40}
+PROTEIN = {'chicken': 32.3, 'turkey': 55, 'ham': 1, 'tofu': 0.1}
+CHEESE = {'cheddar': 5, 'Swiss': 5.5, 'mozzarella': 1000}
+SAUCE = {'mayo': 300, 'mustard': 400, 'lettuce': 500, 'tomato': 600}
+
+sandwich_ingredients = []
+
+
+def sandwich_maker():
+    """Provide information on ordered sandwich/es.
+
+    Based on users input, provides information on:
+    number of ordered sandwiches;
+    sandwich's ingredients;
+    order's price
+    """
+    while True:
+
+        bread_type = pyip.inputMenu(['wheat', 'white', 'sourdough'])
+        sandwich_price = BREAD[bread_type]
+        sandwich_ingredients.append(bread_type)
+
+        protein_type = pyip.inputMenu(['chicken', 'turkey', 'ham', 'tofu'])
+        sandwich_price = PROTEIN[protein_type] + sandwich_price
+        sandwich_ingredients.append(protein_type)
+
+        optional_cheese = pyip.inputYesNo(
+            prompt="Would you like to add cheese?\n",
+            caseSensitive=False)
+        if optional_cheese == "yes":
+            cheese_type = pyip.inputMenu(['cheddar', 'Swiss', 'mozzarella'])
+            sandwich_price = CHEESE[cheese_type] + sandwich_price
+            sandwich_ingredients.append(cheese_type)
+
+        optional_sauce = pyip.inputYesNo(
+            prompt="Would you like to add sauce?\n",
+            caseSensitive=False)
+        if optional_sauce == "yes":
+            sauce_type = pyip.inputMenu(['mayo', 'mustard',
+                                         'lettuce', 'tomato'])
+            sandwich_price = SAUCE[sauce_type] + sandwich_price
+            sandwich_ingredients.append(sauce_type)
+
+        number_of_sandwiches = pyip.inputInt(
+            prompt="How many sandwiches would you like to buy?\n")
+        sandwich_price = "{:.2f}".format(sandwich_price * number_of_sandwiches)
```

Comment:
Formatting of floats can be also done with f-strings although syntax is a bit different:
```
sandwich_price = f"{sandwich_price * number_of_sandwiches:.2f}"
```
Also you are assigning string to `sandwich_price` although it was earlier float. Python allows this due to its dynamic nature but this is usually not the best idea, especially with longer code.

Personally I would introduce new variable `total_price = sandwich_price * number_of_sandwiches` and then format it in f-string in print statement `f"That's a total of {total_price:.2f} kg of gold"`. Of course if you are rich enough to don't care about grams of gold :)

Reply:
`total_price` variable introduced